pom.xml

```diff
@@ -133,6 +172,10 @@
     <thymeleaf-layout-dialect.version>2.3.0</thymeleaf-layout-dialect.version>
 
     <web-security-java.version>1.1.0-rc1</web-security-java.version>
+
+
+    <home.url></home.url>
```

Comment:
Should this be where the other env variables are declared?

Reply:
This is declared here so that it can be referenced by the other Environment variables, like <gettingStarted.url>${home.url}/getting-started</gettingStarted.url>.
I think normally we would prefer to not do this in the pom generally because they aren't contained.

Reply:
I see, it is a bit confusing as it's not contained. However we have the same line on line 32, which looks to be performing the same function.

Reply:
No. This line is declaring home.url to be a POM property, line 32 is saying Set Enviroment Variable 'home.url' to be the same as POM property 'home.url'
I do agree that its a bit confusing though

Reply:
Right makes sense, I'll approve it and maybe later on we can see if there is a less confusing way of implementing it.